src-plugins/reformat/resliceToolBox.cpp

```diff
@@ -166,7 +165,6 @@ void resliceToolBox::startReformat()
         d->reformatedImage = 0;
 
         // close the initial tab which is not needed anymore
-        getWorkspace()->stackedViewContainers()->unlockTabs();
```

Comment:
I may be mistaken here, but if I recall correctly you get a bug if you call removeTab when tabs are locked, which is why there was a call to unlockTabs here. The solution before was to unlock the tabs, remove and lock them back. This should be tested of course.